daml-script/daml/Daml/Script.daml

```diff
@@ -156,8 +157,9 @@ data ScriptF a
 -- (try x catch f) >>= f should not pull `f` in the try block
 data CatchPayload a = CatchPayload
   with
-    act : () -> Free ScriptF (a, ())
-    handle : AnyException -> Optional a
+    act : () -> Free ScriptF (Free ScriptF LedgerValue, ())
```

Comment:
we can simplify this to

```diff
- act : () -> Free ScriptF (Free ScriptF LedgerValue, ())
+ act : () -> Free ScriptF LedgerValue
```

but I'd rather do that separately so the fix is easier to understand